src/main/java/org/rev317/min/debug/DGroundItems.java

```diff
@@ -1,31 +1,49 @@
 package org.rev317.min.debug;
 
+import org.parabot.core.Context;
 import org.parabot.core.paint.AbstractDebugger;
+import org.parabot.core.paint.PaintDebugger;
+import org.parabot.core.ui.Logger;
 import org.rev317.min.api.methods.GroundItems;
+import org.rev317.min.api.methods.Interfaces;
 import org.rev317.min.api.wrappers.GroundItem;
 
 import java.awt.*;
 
 public class DGroundItems extends AbstractDebugger {
 
+    private boolean enabled;
+    private long lastCheck = System.currentTimeMillis();
+    private int cached;
+
     @Override
     public void paint(Graphics g) {
-
+        if (System.currentTimeMillis() - lastCheck > 1000L) {
```

Comment:
Paint will only be reloaded every second anyways, why is this required?

Reply:
I wasn't aware of the tick rate paint() was called so the 1000L was just a precaution against calling `Scene.getObjects()` and generating a 2k size array every few ms. Can remove if not needed

Reply:
Would be great if it was removed, that way we can control the speed from within the abstract layer instead of every class

Reply:
Done